### ISSUE TITLE
Pubkey not allowing to convert to string?

### ISSUE THREAD

**Issue by Morgandri1** [kevinheavey/solders#56]
Getting this error:

``AttributeError: 'solders.pubkey.Pubkey' object has no attribute 'string'``

when my code is:
```
from solders.keypair import Keypair

kp = Keypair()
print(kp.pubkey().string())
```

This error is happening on both 0.14.0 and 0.17.0

**Comment by kevinheavey** [comment]
use `str(key)` not `key.string()`. Did a person or large language model tell you that `key.string()` was a thing? If so please tell at them

**Comment by Morgandri1** [comment]
No, i just assumed it would work seeing as there's a ``string`` method in the pubkey.pyi src. might want to remove that

**Comment by kevinheavey** [comment]
Ah I'll go yell at myself then

**Comment by Morgandri1** [comment]
haha, sorry about that.
Its been in releases since at least 0.14.0, so thats where the confusion was

**Comment by kevinheavey** [comment]
Thanks for pointing it out! Happy to accept a PR to delete it from the pyi file, otherwise will look later this week

**Comment by kevinheavey** [comment]
Closed by #57
